ProjectSettings/ProjectSettings.asset

```diff
@@ -511,7 +511,7 @@ PlayerSettings:
   webGLCompressionFormat: 1
   scriptingDefineSymbols:
     1: 
-    7: 
+    7: LEAP_TESTS
```

Comment:
Hm we need to figure out how to disable leap tests hahah, we don't want to commit with tests enabled